tests/backup/backup_basic_test.go

```diff
@@ -20,7 +20,6 @@ const (
 	clusterName              = "tp-cluster"
 	restoreNamePrefix        = "tp-restore"
 	configMapName            = "kubeconfigs"
-	sourceClusterName        = "source-cluster"
```

Comment:
Are you sure about removing this?

Reply:
In /Users/snigdhaagrawal/automation/torpedo_backup/torpedo/tests/common.go -> SourceClusterName is already declared